### PR TITLE
Modularized process. modified flow of preprocess/process as discussed

### PR DESCRIPTION
**Pull request by jinhyunglee**
also fixed lots of flake8 that it complained.

### CODE REVIEW

File: examples/config_sample.yaml [hunk 1]
```diff
@@ -18,10 +18,8 @@ recordings:
   format: long
 
 clustering:
-  clustering_method: neigh_chan
-  
-location:
-  isolation_threshold: 200
```
Comment (edublancas) [inline examples/config_sample.yaml]: did you remove this entire section? (location)

File: examples/config_sample_complete.yaml [hunk 2]
```diff
@@ -113,7 +108,7 @@ triage:
   # number of nearest neighbors to consider
   nearest_neighbors: 20
   # percentage of data to be triaged
-  percent: 0.01
```
Comment (edublancas) [inline examples/config_sample_complete.yaml]: why did the default values change?

File: src/yass/process/triage.py [hunk 3]
```diff
@@ -1,45 +1,55 @@
...
 import numpy as np
 from scipy.spatial import cKDTree
 
 
-# FIXME: why is there a do_triage parameter on this function?
-# TODO: documentation
-# TODO: comment code, it's not clear what it does
-def triage(score, channel_index, triage_k, triage_percent, do_triage):
-    """[?]
+# FIXME: dotriage should not be here
```
Comment (edublancas) [inline src/yass/process/triage.py]: i see that you removed the `doTriage`, can you please remove the FIXME comment?

File: examples/config_sample.yaml (hunk 1 above)
Comment (jinhyunglee) [reply examples/config_sample.yaml]: yes. this was for isolation triage but we removed the algorithm

File: examples/config_sample_complete.yaml (hunk 2 above)
Comment (jinhyunglee) [reply examples/config_sample_complete.yaml]: 1% was what I was experimenting and somehow got in there as default. 
1% is essentially not doing triage

File: src/yass/process/triage.py (hunk 3 above)
Comment (jinhyunglee) [reply src/yass/process/triage.py]: alright

Comment (jinhyunglee) [reply src/yass/process/triage.py]: Done

**Pull request merged**